versioned_docs/version-6.x.x/getting-started.md

```diff
@@ -44,7 +45,7 @@ const sequelize = new Sequelize({
 // Option 3: Passing parameters separately (other dialects)
 const sequelize = new Sequelize('database', 'username', 'password', {
   host: 'localhost',
-  dialect: /* one of 'mysql' | 'mariadb' | 'postgres' | 'mssql' */
+  dialect: /* one of 'mysql' | 'mariadb' | 'postgres' | 'mssql' | 'oracle' */
```

Comment:
Oh. I think we are actually lacking some others here. Would you mind expanding it to

'mysql' | 'postgres' | 'sqlite' | 'mariadb' | 'mssql' | 'db2' | 'snowflake' | 'oracle'

Reply:
changes done.